tests/test_fileIO.py

```diff
@@ -53,6 +57,22 @@ def test_JSON(self, name, obj):
         newObj = readJSON(self.fileName, comm=self.comm)
         assert_equal(obj, newObj)
 
+    @patch('sys.stdout', new_callable=io.StringIO)
+    @patch('sys.stderr', new_callable=io.StringIO)
+    def test_redirectingIO(self, mock_stdout, mock_stderr):
+        mock_stdout.fileno = MagicMock(return_value=0)
+        mock_stderr.fileno = MagicMock(return_value=1)
+
+        self.fileName = 'test_redirect.out'
+
+        for i in range(2048):
+            with redirectingIO(open(self.fileName, 'a')):
+                print(f'test_{i}')
+
+        # This is only here so the test is considered ok if
+        # it finishes without error
+        self.assert_(True)
```

Comment:
Why did you use `self.assert_`? I don't think that's in the public API right? Maybe `self.assertTrue(True)`?

Reply:
Basically laziness. I found an example that was like this and used it without thinking. I changed it.